lib/src/platform_text_field.dart

```diff
@@ -377,6 +427,7 @@ class PlatformTextField
     this.autofillHints,
     this.cursorHeight,
     this.restorationId,
+    this.hint,
```

Comment:
Please name this `hintText`. The variable names through out the package tries to match the `material` version as much as possible.